_data/changelogs/accordion.yml

```diff
@@ -0,0 +1,36 @@
+title: Accordion
+type: Component
+changelogURL:
+items:
+  - date: 2022-08-05
+    summary: Added <code>type="button"</code> to the component code and guidance.
+    additional_info:
+    isBreaking: 'Potential'
+    isCodeUpdate: true
```

Comment:
It looks like these aren't used on the accordion component page, is that intentional? It would be nice to see this information.

Reply:
I am not sure I understand. I noticed that I had forgotten to update the `changelog-table.html` file to reflect some of the new data keys. It should be displaying all the data now. Is this what you are referring to?

Reply:
Okay, I just double-checked and am seeing them now.

![image](https://user-images.githubusercontent.com/3385219/198144719-d6e80121-233b-491b-ab48-1913e061a60b.png)